development/tools/cmd/pjtester/README.md

```diff
@@ -14,12 +14,14 @@ It is configured by the **run-if-changed** option:
 run_if_changed: "^(vpath/pjtester.yaml)"
 ```
 
-`pjtester` expects to find the the configuration of the ProwJobs to tests under `vpath/pjtester.yaml`.
+`pjtester` expects to find the configuration of the ProwJobs to tests under `vpath/pjtester.yaml`.
 
 By default, `pjtester` disables all reporting for the ProwJob. That means no Slack messages and no status report on Github. To check the test results, consult the [ProwStatus](https://status.build.kyma-project.io/) dashboard.
 
 Details from `pjtester.yaml` and from the ProwJob environment variables are used to construct the specification of the ProwJob to test. `pjtester` uses the environment variables created by Prow for the presubmit job which identify the pull request and its commit hash in the `test-infra` repository. The generated ProwJob to test then uses the `test-infra` code from the pull request's head, ensuring that the latest code is under test.
 
+For presubmit jobs, prow require to have pull request head SHA, pull request number and pull request author set in prowjob refs. If you don't provide in pjtester.yaml file a pull request number for repo against which tested prowjob is running, `pjtester` will find pull request for master branch head and use this details for presubmit refs.
```

Comment:
```suggestion
For presubmit jobs, Prow requires the pull request's head SHA, pull request number, and pull request author set in the ProwJob refs. In the `pjtester.yaml file`, you must specify a pull request number for a repository against which a tested Prow job is running. If you don't specify one, `pjtester` will find a pull request for the `master` branch (`HEAD`) and use its details for the presubmit refs.
```

Reply:
`pjtester` is the name of the tool and I don't think it should not be in code font, but I can see it is written this way all over the doc so I left it as it is.